docs: JUnit Processing and Visualization (#127)
kubeshop/testkube-docs: docs/articles/results-overview.md

> - [Logs & Artifacts](logs-and-artifacts) provide the foundation for test execution troubleshooting

Reviewer: ```suggestion
- [Logs & Artifacts](logs-and-artifacts) provide the foundation for test execution troubleshooting.
```

> - [Log Highlighting](../testkube-pro/articles/log-highlighting) makes it easy to quickly identify key findings in test execution logs

Reviewer: ```suggestion
- [Log Highlighting](../testkube-pro/articles/log-highlighting) makes it easy to quickly identify key findings in test execution logs.
```

> - [AI Analysis](../testkube-pro/articles/AI-test-insights) uses AI to analyse log outputs and pinpount key findings

Reviewer: ```suggestion
- [AI Analysis](../testkube-pro/articles/AI-test-insights) uses AI to analyze log outputs and pinpoint key findings.
```

> - [Cached Test Results](../testkube-pro/articles/cached-results) help you work with execution results even if the cluster your tests were run in is not available

Reviewer: ```suggestion
- [Cached Test Results](../testkube-pro/articles/cached-results) help you work with execution results even if the cluster your tests were run in is not available.
```
